src/components/Person/Person.jsx

```diff
@@ -1 +1,27 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export function Person({
+  person: {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  },
+}) {
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${name}`}</h2>
+      {age ? (
+        <p className="Person__age">{`I am ${age}`}</p>
+      ) : null}
+      {isMarried ? (
+        <p className="Person__partner">
+          {`${partnerName} is my ${sex === 'm' ? 'wife' : 'husband'}`}
+        </p>
+      ) : (
+        <p className="Person__partner">I am not married</p>
+      )}
```

Comment:
1 - you can just select needed text, these paragraphs are similar
2 - better to save this `wife/husband` string to constant and use here, nested ternary operators are not so readable